### DISCUSSION TITLE
Filemanager and NTFS File System

### DISCUSSION THREAD

**Discussion by ozapft** [aldostools/webMAN-MOD#681] [Q&A] [answered]
Only a small question. 
Today I tried to copy a ISO file with the file manager of webMAN MOD on a NTFS Drive from one directory to another.
I could mark it and paste it.
The process took only fractions of a second but despite a successful message nothing was copied. It should actually take minutes.
Is a NTFS Drive read only with webMAN MOD? 

I usually use irisman to copy files on NTFS drives.

**Comment by aldostools** [comment, marked as answer]
After review this feature, for some reason it was broken. I fixed the bug in this commit 9a0a32d4b95914528b88fe012f79f545383d155b

**Reply by ozapft** [reply]
Thank you for your answer.
I´ve tested the new version but it freeze the PS3 while pressing _paste_
Copy and paste with Fat32 is working.

**Reply by aldostools** [reply]
> Thank you for your answer. I´ve tested the new version but it freeze the PS3 while pressing _paste_ Copy and paste with Fat32 is working.

Thank you for your feedback. The it should freeze the same if your copy NTFS files using the /copy.ps3 command.

/cpy.ps3&lt;source-path> basically stores the source file in a temporary buffer.
/paste.ps3&lt;dest-path> builds the command /copy.ps3&lt;source-path>|&lt;dest-path> and executes it.

NTFS library seems to use too much memory while it's copying files. I will try to reduce the copy buffer from 256KB to 128KB.

**Reply by ozapft** [reply]
/cpy.ps3 and /paste.ps3  - **tested** with my raspberry pi. hdd0 to NTFS **-OK**
/copy.ps3 - **tested over network OK**
I only tested this with a small icon0.png

**Update: With a bigger file it freezes too!**

BTW: for the dev **/dev_hdd0/** I need no ":" after hdd0 but for dev_ntfs0 I have to write **/dev_ntfs0:/**

**Reply by aldostools** [reply]
I will upload a new build later today with less memory usage. The ":" is required by the ntfs lib. I don't know why the devs of that library used that format.
